src/pages/VeSDL/VeTokenCalculator.tsx

```diff
@@ -122,15 +122,18 @@ export default function VeTokenCalculator({
             select
             SelectProps={{ IconComponent: ArrowDownIcon }}
           >
-            <MenuItem value="">
-              <em>None</em>
-            </MenuItem>
             {basicPools &&
-              Object.keys(basicPools)?.map((poolName) => (
-                <MenuItem key={poolName} value={poolName}>
-                  {poolName}
-                </MenuItem>
-              ))}
+              Object.keys(basicPools)?.map((poolName) => {
+                const pool = basicPools[poolName]
+                const outdated =
```

Comment:
instead of `return null` can you use .filter before the `.map` to filter out entries that don't belong?

Reply:
OK, Good.